### ISSUE TITLE
Fatal error:  Uncaught Error: Call to undefined function add_query_var()

### ISSUE THREAD

**Issue by galbaras** [nosilver4u/ewww-image-optimizer#190]
**Describe the bug**

```
Fatal error:  Uncaught Error: Call to undefined function add_query_var() in [...]/wp-content/plugins/ewww-image-optimizer/common.php:1955
Stack trace:
#0 [...]/wp-includes/class-wp-hook.php(287): ewww_image_optimizer_notice_reoptimization('')
#1 [...]/wp-includes/class-wp-hook.php(311): WP_Hook-&gt;apply_filters(NULL, Array)
#2 [...]/wp-includes/plugin.php(478): WP_Hook-&gt;do_action(Array)
#3 [...]/wp-admin/admin-header.php(281): do_action('admin_notices')
#4 [...]/wp-admin/index.php(111): require_once('[...]')
#5 {main}
thrown in /home/gboldev/public_html/ronitbaras/wp-content/plugins/ewww-image-optimizer/common.php on line 1955
```

**To Reproduce**

1. Log into site's dashboard
2. See error

**Expected behavior**

No error ;P

**Additional context**

This only seems to be happening on one of my tests sites after updating to v5.6.1. Other tests sites seems fine, despite all running the same image handling plugins.

I've tried deactivating Media File Renamer and Image Resize & Select Crop, and I've re-installed EWWW Image Optimizer, but none of these helped.

**Comment by mundschenk-at** [comment]
I've noticed this as well since the update.

**Comment by mundschenk-at** [comment]
I'm curious where this function should be defined. It does not exist in WordPress (though there is the instance method `WP::add_query_var`).

**Comment by nosilver4u** [comment]
It should actually be add_query_arg(), and it snuck in a couple releases ago, but only triggers on sites where a bunch of images have been re-optimized 5x or more.
Feel free to manually change the code until I can get a bugfix out!

**Comment by nosilver4u** [comment]
fix released as 5.6.2!
